### ISSUE TITLE
Does not contain bitcode

### ISSUE THREAD

**Issue by jonigl** [kaltura/player-sdk-native-ios#133]
Hi,

I have been triying to follow the Getting Started but I have this error when I tried to build and run in my iphone 6 plus (9.2.1): 

> ld: '/Users/jlowenstern/Library/Developer/Xcode/DerivedData/kalturaPlayer-bunyqdblebkjbnbbzrargojgltxp/Build/Products/Debug-iphoneos/libKALTURAPlayerSDK.a(WVURLProtocol.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture arm64
> clang: error: linker command failed with exit code 1 (use -v to see invocation)

*I'm using the player-sdk-native-ios-2.2.3

¿Is there any sample app using this SDK?

What is the problem? Any ideas? I will appreciate them! Thanks in advance!

**Comment by kaltura-hooks** [comment]
Hi @jonigl,

Thank for you reporting an issue and helping to improve Kaltura!

If you're having some issues getting your player set up, there are quite a few steps that can be taken to figure things out. 


In order to expedite the reporting process, please follow instructions in the doc below, prior to submitting a bug/issue

https://github.com/kaltura/player-sdk-native-ios/wiki/How-To-Report-an-Issue-with-the-Kaltura-iOS-Player-SDK


If you only have a general question rather than a bug report, please close this issue, post at:
https://forum.kaltura.org/c/community-forums/player-sdk

And add a comment to the Github issue linking to your post.

Thank you in advance!

**Comment by ElizaSapir** [comment]
Hey @jonigl,

This issue is environmental issue of your app, here is what you supposed to do:
Be sure that you are dealing with Xcode 7, NOT lower versions. In lower version this flag does not exist.

Setup: Project>Build Settings>All>Build Options>Enable Bitcode = NO

![alt tag](http://i.stack.imgur.com/yHSWb.png)

**Comment by jonigl** [comment]
Hi @ElizaSapir thanks for your quickly response!

I did what you told me, I set Enable Bitcode = NO, but when I did that and tried to build and run a received several errors: 

> Undefined symbols for architecture arm64:
>   "std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::replace(unsigned long, unsigned long, char const*, unsigned long)", referenced from:
>       WidevineMediaKit::HTTPHeader::ReplaceChunk(unsigned int, unsigned int, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >) in libKALTURAPlayerSDK.a(HTTPHeader.o)
>   "typeinfo for unsigned char", referenced from:
>       unsigned char const* boost::any_cast<unsigned char const>(boost::any*) in libKALTURAPlayerSDK.a(Mpeg2PsParser.o)
>       boost::any::holder<unsigned char>::type() const in libKALTURAPlayerSDK.a(HTTPDecrypter.o)
>   "typeinfo for unsigned short", referenced from:
>       unsigned short const* boost::any_cast<unsigned short const>(boost::any*) in libKALTURAPlayerSDK.a(Mpeg2PsParser.o)
>       boost::any::holder<unsigned short>::type() const in libKALTURAPlayerSDK.a(HTTPDecrypter.o)
>   "std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::insert(unsigned long, char const*, unsigned long)", referenced from:
>       DataStore::DeleteAllFiles(char const*) in libKALTURAPlayerSDK.a(DataStoreAndroid.o)
>   "std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::compare(unsigned long, unsigned long, char const*, unsigned long) const", referenced from:
>       WV::URL::GetOption(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, bool*) in libKALTURAPlayerSDK.a(PilUrl.o)
>       DataStore::FindFile(void**, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in libKALTURAPlayerSDK.a(DataStoreAndroid.o)
>   "std::__1::basic_ostream<char, std::__1::char_traits<char> >::operator<<(long long)", referenced from:
>       WidevineMediaKit::Pump::Expired() in libKALTURAPlayerSDK.a(Pump.o)
>   "non-virtual thunk to std::__1::basic_iostream<char, std::__1::char_traits<char> >::~basic_iostream()", referenced from:
>       construction vtable for std::__1::basic_iostream<char, std::__1::char_traits<char> >-in-std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> > in libKALTURAPlayerSDK.a(LiveHttpSegmenter.o)
>   "non-virtual thunk to std::__1::basic_iostream<char, std::__1::char_traits<char> >::~basic_iostream()", referenced from:
>       construction vtable for std::__1::basic_iostream<char, std::__1::char_traits<char> >-in-std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> > in libKALTURAPlayerSDK.a(LiveHttpSegmenter.o)
> 

Here the complete output: [link](http://pastebin.com/nnWxScXC)

Do you have any ideas? Thanks a lot!!!

**Comment by jonigl** [comment]
Sorry @ElizaSapir it's me again... I tried with the "SDK CocoaPods Installation" and then I set Enable Bitcode = NO and works lilke a charm! So, perhaps it is interesting to know whats the problem using hte "SDK Traditional Installation" but for now I'm not more stuck and I can go forward with my development.

If you want to follow this inconvenient I will help you, perhaps this is useful to modified the README.md for better Getting Started! :)

**Comment by ElizaSapir** [comment]
Thanks @jonigl , it's in our plans to improve docs :)

About the errors you got with "SDK Traditional Installation" I think you missed the following:
https://github.com/kaltura/player-sdk-native-ios#required-frameworks

Best,
Eliza

**Comment by jonigl** [comment]
You are right @ElizaSapir! :) I missed "Required Frameworks" because I thought that belong to "Linking GoogleAds" topic. 

I added every listed framework but I found that now the extension for some files was changed in Xcode 7 from "dylib" to "tbd". (I learned about this with this: [link](http://stackoverflow.com/questions/31450690/why-xcode-7-shows-tbd-instead-of-dylib)) another thing perhaps to explain in the README.md.

Also I have to follow this step: https://github.com/kaltura/player-sdk-native-ios#adding-resources-bundle missed to because I thought that belong to "Linking GoogleAds" topic. 

And it was important to know that if I use Xcode 7 it is necessary to change Enable Bitcode option.

If you are agree I will modified a little this README.md for a better getting started in the next day by a pull request :)

Thank you for your help was very useful!

Regards,
Jonathan
